### PR TITLE
Peacekeepers Episode Two: Return of the Civil Disputes Officer

### PR DESCRIPTION
**Pull request by Gandalf2k15**
## About The Pull Request

After great deliberation, we have decided to readd the CDO! 

Policy addendum: As a civil disputes officer you are expected to deal with low-level crimes such as minor theft and general complaints from crew. Make extensive use of citations! You are not permitted to carry lethal weapons below red, nor are you permitted to attend high-level threats unless the threat level is above blue. You must keep basic order amongst the crew at all times! Afterall, you're fresh out the academy.

Key facts:
They do not get access to lockers.
They do not get a proper gun.
They do not get much armor at all.
They have a pepperball gun.
They do not have a bowman.

Their basic loadout:
1x pepperball gunset
1x imrpovised gripper gloves
1x flash
2x cableties

image

They report to the HoS, and any officers.

## Why its good for game
Encourages a Wider Range of Roleplaying Scenarios:
The introduction of Civil Disputes Officers has opened up new avenues for roleplaying, allowing players to engage in a variety of scenarios that may not have been possible or relevant with Security Officers alone. Players can now roleplay as shopkeepers, bar patrons, or any other citizen who may be involved in minor disputes or misdemeanors. This increased diversity in roleplaying opportunities provides a richer and more immersive experience for all players on the server.

Enhances Character Development:
The presence of Civil Disputes Officers creates a more realistic and dynamic environment within the game, allowing players to develop their characters more thoroughly. The addition of this role helps to flesh out the game's ecosystem, as it introduces a new layer of complexity and realism to the interactions between different characters. Players can now create characters with more nuanced backgrounds and motivations, knowing that there is a dedicated role for dealing with minor issues that may arise.

Reduces Workload for Security Officers:
By focusing on low-level crimes, Civil Disputes Officers help to ease the workload of Security Officers, allowing them to concentrate on more high-chaos events and dangerous situations. This division of labor not only enhances the realism of the game, but also improves overall server balance and gameplay, as it ensures that each role is better equipped to handle their respective responsibilities.

Promotes Positive Interactions Between Players:
The addition of Civil Disputes Officers to the game encourages more cooperative and positive interactions between players. By focusing on resolving minor disputes, these officers help to foster a sense of community within the game, as players are more likely to engage in dialogue and work together to address problems. This increased level of collaboration and interaction can lead to more enjoyable and fulfilling roleplaying experiences for everyone involved.

Expands Opportunities for New Players:
The role of Civil Disputes Officer offers an excellent entry point for new players who may be interested in learning the ropes of SS13., As Junior Officers, these players have the opportunity to learn the mechanics and dynamics of the game at a more manageable pace, without being immediately thrust into high-stakes situations. This provides a more accessible and enjoyable experience for newcomers, helping to grow and maintain the game's player base.

Enables low-level crimes to exist:
The Civil Disputes Officer is specifically assigned to handle minor offenses, such as theft, minor assault, and various other low-level crimes. By creating a dedicated role for these matters, the game shifts the focus from security dealing exclusively with major threats to also addressing roleplay-centric issues that arise organically. Since Civil Disputes Officers are clearly equipped and trained to handle only low-level crimes, and they must adhere to Standard Operating Procedures (SOP), they are inherently motivated to address these petty offenses effectively.

## Changelog

:cl:
add: The civil disputes officer has made an ultimate comeback!
/:cl:

### CODE REVIEW

**Comment by Nerev4r** [comment]
So, 3 officers, 2 CDOs?

If so, that's okay numbers. If it's 2 CDOs and an apparent 6th seccie, that's a problem; we run 5 officer slots now.

**Comment by YakumoChen** [comment]
Isn't the new security policy already entirely centered around civil disputes? I actually don't feel we need Security Minus now, we've already moved away from sec being only unga baton and brig man.

**Review by GoldenAlpharex — CHANGES_REQUESTED** [review]
They should be getting their sec HUD icon back, no empty sec HUD icons please.

**Comment by desertmote** [comment]
pr description doesn't convince me at all this is going to result in improved roleplaying.

**Comment by Iamgoofball** [comment]
CDOs really would be better off as some form of auto-militia method the HOS can enact that lets him quickly deputize people, not as it's own proper job slot. I had a draft for this at some point but never deployed it. Maybe I should do goofsec 3.

**Comment by SpaceVampire** [comment]
>As stated above, this should replace 50% of the security jobslots meaning 2 Civil Dispute Officer and 5 full seccies.

We are currently running five security officers, so what exactly does this replace?
I'm a little confused by your math here.
Don't department guards more-or-less fill this role already anyway?

**Comment by Gandalf2k15** [comment]
> > As stated above, this should replace 50% of the security jobslots meaning 2 Civil Dispute Officer and 5 full seccies.
> 
> We are currently running five security officers, so what exactly does this replace? I'm a little confused by your math here. Don't department guards more-or-less fill this role already anyway?

the PR body was a copy/paste of the original

**Comment by Gandalf2k15** [comment]
> They should be getting their sec HUD icon back, no empty sec HUD icons please.

oops fixed

**Review by Ebin-Halcyon — CHANGES_REQUESTED** [review]
Use the black softcap as a base for the hat sprites, obj and mob.

Everything else needs to be improved as well but i'll go into that at a later date.

**Comment by Gandalf2k15** [comment]
> Use the black softcap as a base for the hat sprites, obj and mob.
> 
> Everything else needs to be improved as well but i'll go into that at a later date.

After discussion, we've replaced the uniform with the police variant.

**Comment by desertmote** [comment]
Are you rewriting the 'How This Contributes To The Skyrat Roleplay Experience' header? Its missing from the description right now

File: modular_skyrat/modules/junior_officer/junior_officer.dm [hunk 1]
```diff
@@ -53,7 +53,14 @@
 
 	suit = /obj/item/clothing/suit/armor/vest/peacekeeper/brit
 
-	backpack_contents = list(/obj/item/clothing/gloves/tackler/offbrand, /obj/item/storage/box/gunset/pepperball, /obj/item/restraints/handcuffs/cable = 2, /obj/item/modular_computer/pda/security)
+	backpack_contents = list(
+		/obj/item/clothing/gloves/tackler/offbrand,
+		/obj/item/storage/box/gunset/pepperball,
+		/obj/item/restraints/handcuffs/cable = 2,
+		/obj/item/modular_computer/pda/security,
+		/obj/item/paper = 2,
```
Comment (YakumoChen) [inline modular_skyrat/modules/junior_officer/junior_officer.dm]: Two single sheets of paper?
Why not just give them a clipboard or a full paper sheaf

File: modular_skyrat/modules/junior_officer/junior_officer.dm [hunk 2]
```diff
@@ -53,7 +53,14 @@
...
+		/obj/item/pen,
```
Comment (YakumoChen) [inline modular_skyrat/modules/junior_officer/junior_officer.dm]: There is a pen in their PDA already

**Comment by dawsonkeyes** [comment]
I don’t see how this contributes to a more amicable roleplay experience on the server. Shouldn’t the PR at least try to make a point on that matter?

**Comment by Gandalf2k15** [comment]
> I don’t see how this contributes to a more amicable roleplay experience on the server. Shouldn’t the PR at least try to make a point on that matter?

Github is for technical review only, please see discord.

**Comment by softcerv** [comment]
> > I don’t see how this contributes to a more amicable roleplay experience on the server. Shouldn’t the PR at least try to make a point on that matter?
> 
> Github is for technical review only, please see discord.

Don't pull requests on this repository require an explanation as to how they contribute to the Skyrat roleplaying experience? I feel like this is a reasonable question to ask considering our modern pull request guidelines.

**Comment by GoldenAlpharex** [comment]
> > They should be getting their sec HUD icon back, no empty sec HUD icons please.
> 
> oops fixed

The Sec HUD icon still doesn't seem to appear in-game, is it named properly?

**Comment by GoldenAlpharex** [comment]
Unless the server wasn't updated since, which I don't think it was, actually.

**Comment by GoldenAlpharex** [comment]
CDO isn't in the proper category in the Crew Monitor (and possibly the Crew Manifest too)

**Comment by Kepteyn** [comment]
The server usually lacks enough high-level and low-level crime to justify splitting the role into two parts. This division limits the already limited tasks a security officer can do further. If antagonists frequently provided any sort of challenge that required more than five minutes attention to resolve, or the OPFOR system was an effective method of producing sufficient levels of conflict - which, while neither inherently a good or a bad thing, has resulted far, far fewer opposing forces to security - the addition of a role to focus on low-level crime while other officers are dealing with threats might be useful. As it stands, security officers are lacking things to do as is and reducing it by splitting roles is not helpful in my consideration.

**Comment by Gandalf2k15** [comment]
> CDO isn't in the proper category in the Crew Monitor (and possibly the Crew Manifest too)

will address

**Comment by QuacksQ** [comment]
ID and Sechud overlay are missing both missing Icons, image hof the PR also shows it missing.

**Comment by Gandalf2k15** [comment]
> ID and Sechud overlay are missing both missing Icons, image hof the PR also shows it missing.

both are definitely there but the server hasnt been updated

**Comment by Ebin-Halcyon** [comment]
> > Use the black softcap as a base for the hat sprites, obj and mob.
> > Everything else needs to be improved as well but i'll go into that at a later date.
> 
> After discussion, we've replaced the uniform with the police variant.

Remove the unused hats and uniform then.

**Comment by QuacksQ** [comment]
image

Low level crimes already existed in corp regs prior to this role but nobody was committing them, I dont think that adding a role is going to make people commit more crime lol

**Comment by QuacksQ** [comment]
image

Missing Icon

**Comment by YakumoChen** [comment]
still seems to be missing ingame icons

**Pull request closed without merging**